app/tests/test_sms.py

```diff
@@ -198,498 +199,473 @@ def test_get_info(self):
         self.assertEqual(1, Client.query.count())
         self.assertEqual(1, Event.query.count())
         self.assert_twilio_response(
-            "hazebot runs on PurpleAir sensor data and is a free service providing accessible local air quality information. "
```

Comment:
Yeah this file is pretty F'd. You need to revert the indentation since these are methods on the class. (Indent one more tab.) See if that causes tests to pass?